Makefile

```diff
@@ -2,21 +2,61 @@
 build-and-push-daemon:
 	bash scripts/build-and-push-daemon.sh
 
+# dev-testnet
+.PHONY: build-daemon-dev-testnet
+build-daemon-dev-testnet:
+	bash scripts/build-daemon.sh dev-testnet
+
+.PHONY: push-daemon-dev-testnet
+push-daemon-dev-testnet:
+	bash scripts/push-daemon.sh
+
 .PHONY: deploy-lambdas-dev-testnet
 deploy-lambdas-dev-testnet:
 	yarn workspace wallet-service run serverless deploy --stage dev-testnet --region eu-central-1
```

Comment:
At some point, I believe we had renamed  this one into `ci-deploy-lambdas-dev-testnet`, and created a new one like this:

```
.PHONY: deploy-lambdas-dev-testnet
deploy-lambdas-dev-testnet:
	   AWS_SDK_LOAD_CONFIG=1 yarn workspace wallet-service run serverless deploy --stage dev-testnet --region eu-central-1
```

Without the env var, the deploy doesn't work from a local machine using `aws sso` to authenticate, right?

Should we do this again?

Reply:
We should, done!

Thanks